src/screens/SubjectSetsIndex/SubjectSetPageContainer.js

```diff
@@ -12,21 +12,21 @@ function SubjectSetPageContainer({ history, match }) {
   const store = React.useContext(AppContext)
 
   React.useEffect(() => {
-    store.groups.selectGroup(null)
-  }, [store])
+    store.getResources(match.params)
+  }, [match, store])
 
   const onSelection = group => {
-    store.groups.selectGroup(group)
-    const nextPath = generatePath(SUBJECTS_PATH, { subjectSet: group.id, ...match.params})
+    const nextPath = generatePath(SUBJECTS_PATH, { group: group.display_name, ...match.params})
     history.push(nextPath)
   }
+  const groups = Array.from(store.groups.all.values())
 
   return (
     <Box margin='medium' fill='vertical'>
       <ResourcesTable
         columns={columns}
-        data={store.groups.all}
-        resource='Subject Sets'
+        data={groups}
+        resource='groups'
```

Comment:
It seems that most of the text labels and class names that use "Subject Set" (or "Set") have been renamed to "Group", which is cool.

The following still seem to be called "Subject Set" though:
- src/screens/SubjectSetsIndex
- src/screens/SubjectSetsIndex/SubjectSetPageContainer
- blablabla/HeaderButton/DownloadSetDataContainer

Reply:
Good point. Fixed.